src/libs/ajax/leonardo/models/runtime-models.ts

```diff
@@ -67,6 +82,9 @@ export interface ListRuntimeItem {
   patchInProgress: boolean;
 }
 
+export const sanitizeListRuntime = (listRuntime: ListRuntimeItem): SanitizedListRuntimeItem =>
+  _.omitBy((value, key) => key === 'workspaceId' && value === null, listRuntime) as SanitizedListRuntimeItem;
```

Comment:
Hmm... can we omit all null values instead?

i.e.
`_.omitBy(listRuntime, v => v === null);`

Reply:
It would be nice to have stronger type safety in this method instead of relying on a cast.

Reply:
agreed Nick, I would copy known properly only, which is a bit more code to type out, but:
- is safer
- removes the need for magic knowledge of things in other areas of the code,
- avoid overly general stripping code (later they may be legit properties with value null)
- and should avoid needing to "as MyType" cast at all.

Reply:
@msilva-broad what's your preferred approach for copying known properties only?

Reply:
Got it! That makes sense.